docs/_posts/HashamUlHaq/2021-11-05-ner_covid_trials_en.md

```diff
@@ -0,0 +1,191 @@
+---
+layout: model
+title: Extract entities in covid trials
+author: John Snow Labs
+name: ner_covid_trials
+date: 2021-11-05
+tags: [ner, en, clinical, licensed]
+task: Named Entity Recognition
+language: en
+edition: Spark NLP for Healthcare 3.2.3
```

Comment:
Shouldn't this be 3.3.2, or at least 3.3.1?

Reply:
it should be 3.3.2